internal/testhelper/registry.go

```diff
@@ -0,0 +1,94 @@
+package testhelper
+
+import (
+	"fmt"
+
+	"github.com/keep-network/keep-common/pkg/persistence"
+	"github.com/keep-network/keep-ecdsa/pkg/ecdsa/tss"
+)
+
+type PersistenceHandleMock struct {
+	PersistedGroups  []*TestFileInfo
+	Snapshots        []*TestFileInfo
+	ArchivedGroups   []string
+	outputDataChan   chan persistence.DataDescriptor
+	outputErrorsChan chan error
+}
+
+func NewPersistenceHandleMock(outputDataChanSize int) *PersistenceHandleMock {
+	return &PersistenceHandleMock{
+		outputDataChan:   make(chan persistence.DataDescriptor, outputDataChanSize),
+		outputErrorsChan: make(chan error),
+	}
+}
+
+type TestFileInfo struct {
+	Data      []byte
+	Directory string
+	Name      string
+}
+
+func (phm *PersistenceHandleMock) Save(data []byte, directory string, name string) error {
```

Comment:
I think the linter will yell at you unless you give all of these public structs / functions comments

Reply:
😱 great point!